composer.json

```diff
@@ -14,7 +14,7 @@
     "php-http/httplug": "^1.1"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7",
+    "phpunit/phpunit": "^7.5",
```

Comment:
I believe we went to great lengths to be able to still support php 5. Now that we require `phpunit 7.5` I believe we automatically require `php: ^7.1`

Reply:
I did not alter `"php": ">=5.6"` so this only affects development.

However, I see that CircleCI is now running 4 tests on `php72` so with this update we cannot test on the php 5.6 container anymore. Let's discuss on monday.

Reply:
I reverted the phpunit dependency.